kuronometer-core/src/test/scala/com/github/pedrovgs/kuronometer/generators/ConfigGenerators.scala

```diff
@@ -0,0 +1,17 @@
+package com.github.pedrovgs.kuronometer.generators
+
+import com.github.pedrovgs.kuronometer.free.domain.{Config, Platform}
+import org.scalacheck.{Arbitrary, Gen}
+
+object ConfigGenerators {
+
+  implicit lazy val arbConfig: Arbitrary[Config] = Arbitrary(config())
```

Comment:
This probably does not need to be lazy since Arbitraries are essentially functions and won't do anything except the one allocation until they run or you invoke one of their combinators.